### PR TITLE
Complete credits

### PR DESCRIPTION
**Pull request by Kermalis**

### CODE REVIEW

File: src/credits.c [hunk 1]
```diff
@@ -1,39 +1,180 @@
...
+                    *destText2 = curChar;
+                    srcText++;
+                    destText2++;
+                }
+                *destText2 = 0;
+                srcText = buffer;
+                break;
+        }
+
+        xxx_call_draw_string(temp2, counter, srcText, 0, 0);
```
Comment (SethBarberee) [inline src/credits.c:111]: `xxx_call_draw_string` is `(s32 x, s32 y, u8 *text, u32, u32)` so: 

`temp2` -> x (coord) so `unk2` is x coord of that credit

`counter` -> y (coord) so `unk4` is the y offset of that credit

File: src/credits.c [hunk 2]
```diff
@@ -1,39 +1,180 @@
...
+        int b = 9;
+        unsigned long long c = b;
+        sub_8099AFC(c, 0, a[0]);
+    } while(0);
+    return 1;
+}
+
+s32 sub_8035574(void)
+{
+    register u32 const *p asm("r2");
```
Comment (SethBarberee) [inline src/credits.c:128]: Surround in `NONMATCHING`

**Review by SethBarberee — CHANGES_REQUESTED** [review]
Looks good aside from two minor things

**Comment by Kermalis** [comment]
Addressed

**Review by SethBarberee — APPROVED** [review]

**Pull request merged**